core/src/main/scalajs/weaponregex/WeaponRegeXJS.scala

```diff
@@ -38,7 +38,7 @@ object WeaponRegeXJS {
     *   A JavaScript Array of [[weaponregex.model.mutation.Mutant]] if can be parsed, or throw an exception otherwise
     */
   @JSExportTopLevel("mutate")
-  def mutate(pattern: String, flags: String, options: MutationOptions): js.Array[MutantJS] = {
+  def mutate(pattern: String, flags: js.UndefOr[String], options: MutationOptions): js.Array[MutantJS] = {
```

Comment:
Should de options be required too? If not then the 'default' API might be a bit awkward:
```js
wrx.mutate('\\u{20}', undefined, {});
// VS
wrx.mutate('\\u{20}');
```

Reply:
I just realized you already answered this question twice 😅. It's because of the overloads. I guess in a future version when the deprecated export is removed we can make `options` optional